scripts/ci/import_signing.sh

```
@@ -0,0 +1,111 @@
+  if [[ -n "$parsed" ]]; then
+    ORIG_KEYCHAIN_ARRAY+=("$parsed")
+  fi
+done < <(security list-keychains -d user)
+ORIG_DEFAULT_KEYCHAIN=$(security default-keychain | tr -d '"')
+ORIG_KEYCHAINS_JOINED=""
+if (( ${#ORIG_KEYCHAIN_ARRAY[@]} > 0 )); then
+  for keychain in "${ORIG_KEYCHAIN_ARRAY[@]}"; do
+    ORIG_KEYCHAINS_JOINED+="$keychain"$'\n'
+  done
+fi
+ORIG_KEYCHAINS_B64=$(printf '%s' "$ORIG_KEYCHAINS_JOINED" | python3 - <<'PY'
```

Shellcheck (0.11.0)

[error] This redirection overrides piped input. To use both, merge or pass filenames.

(SC2259)